libs/xmlrpc-c/patches/001-fix-pkg-config-files.patch

```diff
@@ -0,0 +1,327 @@
+Index: lib/abyss/src/Makefile
+===================================================================
+diff --git a/lib/abyss/src/Makefile b/lib/abyss/src/Makefile
+--- a/lib/abyss/src/Makefile	(revision 3202)
++++ b/lib/abyss/src/Makefile	(revision 3205)
```

Comment:
- Who is the author?
- Where is the commit subject, commit description
- Is this going to be upstreamed?

And more questions, I do have...

Reply:
I have updated the PR description describing the origin of the patch. As described in this commit log, this patch is already upstream.